templates/registration/logged_out.html

```diff
@@ -0,0 +1,6 @@
+{% extends "base.html" %}
+
+{% block content %}
+  <p>You have been logged out!</p>
+  <a href="{% url 'login' %}">Click here to connect again :))</a>

```

Comment:
The link to the login page is correctly implemented. Ensure that the `login` URL is correctly configured in your authentication URLs.